science/pymol/Portfile

```diff
@@ -80,44 +67,17 @@ depends_lib-append  port:freetype \
                     port:py${python.version}-numpy \
                     port:py${python.version}-pyqt5
 
-variant x11 {
-    depends_lib-delete  port:py${python.version}-pyqt5
-
-    depends_lib-append  port:freeglut \
-                        port:mesa \
-                        port:py${python.version}-pmw \
-                        port:py${python.version}-tkinter \
-                        port:tcl \
-                        port:tk
-
-    require_active_variants tcl "" corefoundation
-    require_active_variants tk "" quartz
-
-    build.args-append   --glut --no-osx-frameworks
-}
-
 patchfiles          pymol_shell.diff \
-                    pmg_tk_platform.patch \
-                    apbs-psize.patch  \
-                    python_string_split.patch \
-                    pdb2pqr.patch \
                     setup.py.diff
 
 if {${subport} eq "pymol-devel"} {
-    patchfiles-delete \
-                    apbs-psize.patch  \
-                    python_string_split.patch \
-                    pdb2pqr.patch \
     patchfiles-replace \
                     setup.py.diff \
                     patch-devel-setup.py.diff
 }
```

Comment:
the patchfiles should be the same now for both `pymol` and `pymol-devel` so you can remove the special case for the `pymol-devel` port. 

However, the `setup.py.diff` doesn't apply anymore (how did you build this locally?) - you should probably just use rename the `patch-devel-setup.py.diff` to `patch-setup.py.diff` and use that one instead.